examples/with-emailpassword-vercel/src/Home/CallAPIView.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+import Session from "supertokens-auth-react/recipe/session";
+import { getApiDomain } from "../App";
+Session.addAxiosInterceptors(axios);
+
+export default function CallAPIView() {
+    async function callAPIClicked() {
+        // this will also automatically refresh the session if needed
+        let response = await axios.get(getApiDomain() + "/sessioninfo");
```

Comment:
This is the wrong path. It should be querying `getApiDomain() + "/api/sessioninfo"`